src/fetch/fetchWithRetries.js

```diff
@@ -86,7 +86,12 @@ function fetchWithRetries(
             retryRequest();
           } else {
             // Request was not successful, giving up.
-            reject(response);
+            var error: any = new Error(sprintf(
+              'fetchWithRetries(): Gave up after %s re-tries.',
```

Comment:
You mean something like `fetchWithRetries(): Still no successful response after %s re-tries, giving up.` ?